pkgs/development/python-modules/sphinx-jsonschema/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  docutils,
+  requests,
+  jsonpointer,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-jsonschema";
+  version = "1.19.1";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
I recently learned this is preferred even for setuptools packaged libraries.